app/code/Magento/Payment/Helper/Data.php

```diff
@@ -183,7 +176,7 @@ public function getMethodFormBlock(MethodInterface $method, LayoutInterface $lay
      */
     public function getInfoBlock(InfoInterface $info, LayoutInterface $layout = null)
     {
-        $layout = $layout ?: $this->_layout;
+        $layout = $layout ?: $layoutFactory->create();
```

Comment:
The `$layoutFactory` variable won't exist here, you'll need to store it in a private member in the constructor to be able to use it, right?

Reply:
Nevermind, I'm too slow 😆

Reply:
Just realized after clicking on create pr :D